src/core/filesystem.cc

```diff
@@ -541,19 +535,13 @@ GCSFileSystem::ReadTextFile(const std::string& path, std::string* contents)
 }
 
 Status
-GCSFileSystem::DownloadFileFolder(
-    const std::string& path, std::string* local_path)
-{
-  // For GCS we don't actually need to download the folder. We use tensorflow's
-  // built in support for loading models directly from GCS.
```

Comment:
Not sure how I missed this before... we need full support for GCS for all models... we don't use TF's built in support anymore, do we... and that wouldn't work for other frameworks anyway. I'm guessing other use-cases don't work either, like ONNX models that are split across separate files.

Reply:
I will file a ticket for this.